var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -51,7 +51,6 @@ class Paraview(CMakePackage, CudaPackage):
             description='Use module kits')
 
     conflicts('+python', when='+python3')
-    conflicts('+python', when='@5.6:')
```

Comment:
I'd update this to `>=5.9` instead of completely removing it. 5.9 onwards, Python2 is no longer supported.